CMakeLists.txt

```diff
@@ -182,16 +182,11 @@ endif()
 
 set(PROFILES_SCHEMA_FILENAME "profiles-0.8-latest.json")
 
-# The scripts directory regenerates the Profiles source and headers.
-add_subdirectory(scripts)
+# The profiles directory regenerates the Profiles source and headers.
+add_subdirectory(profiles)
```

Comment:
Moving the logic to profiles made the installation code easier to write. And IMO reads better.